src/Compilers/VisualBasic/Test/Syntax/Parser/ParseDirectives.vb

```diff
@@ -2396,28 +2415,48 @@ End Module]]>,
 
     <Fact()>
     Public Sub ParseWarningDirective_LineContinuation4()
+        ' PROTOTYPE LanguageVersion.VisualBasic15_3 should be LanguageVersion.VisualBasic15_5
+        ' PROTOTYPE 15.5 should be 16
         Dim tree = ParseAndVerify(<![CDATA[#Enable Warning bc41007 _ 'Comment]]>,
+                                  New VisualBasicParseOptions(LanguageVersion.VisualBasic15_3),
             <errors>
-                <error id="30196" message="Comma expected." start="24" end="24"/>
-                <error id="30999" message="Line continuation character '_' must be preceded by at least one white space and must be the last character on the line." start="24" end="25"/>
-                <error id="30203" message="Identifier expected." start="34" end="34"/>
+                <error id="37306" message="Please use language version 15.5 or greater to use comments after line continuation character." start="24" end="25"/>
             </errors>)
         tree.VerifyOccurrenceCount(SyntaxKind.EnableWarningDirectiveTrivia, 2)
 
         Dim root = tree.GetRoot()
-        Dim skippedTokens = root.DescendantNodes(descendIntoTrivia:=True).OfType(Of SkippedTokensTriviaSyntax).Single
-        Assert.Equal(SyntaxKind.BadToken, skippedTokens.DescendantTokens.Single.Kind)
+        Dim skippedTokens = root.DescendantNodes(descendIntoTrivia:=True).OfType(Of SkippedTokensTriviaSyntax)
+        Assert.True(skippedTokens.isempty)
```

Comment:
>isempty [](start = 34, length = 7)

nit: strange capitalization for IsEmpty. 
super nit: you could do `Assert.IsEmpty(skippedTokens)` here and elsewhere.

Reply:
I think it is Assert.Empty and I changed to that.